### PR TITLE
Hotfix/2.3.0

### PR DESCRIPTION
**Pull request by akueisara**
- [x] Change the get-size endpoint from `get-size-new` to `get-size`
- [x] Prevent from updating the selected user product ID when the event `user-added-product` is fired 
- [x] Add the MIT license
- [x] Remove the `Proxima Nova` font and use the system font
- [x] Add the CHANGELOG.md
- [x] Improve the integration by binding each Virtusize widget with a `VirtusizeProduct` object using the `Virtusize.setVirtusizeView` function
- [x] Update the value of `virtusizeParams.externalUserId` in each attached Virtusize widget on the app when `Virtusize.setUserId` is called
- [x] Fix unit tests

### CODE REVIEW

File: README-JP.md [hunk 1]
```diff
@@ -177,30 +177,36 @@ Proguardをお使いの場合、Proguardのルールファイルに下記のル
 
 ### 2. 商品詳細をセットする
 
-1. アクティビティ内では、比較ビューに反映させるために商品の `imageUrl` と、API で商品を参照するために使用する `externalId` を渡して、商品の詳細を設定します。
+1. アクティビティ内では、
 
+   - Create a `VirtusizeProduct` object with:
+  - An `exernalId` that will be used to reference the product in the Virtusize server
+     - An `imageURL` for the product image
```
Comment (gpark-vs) [inline README-JP.md:184]: Misalignment of two lines..? 🤔

File: README-JP.md [hunk 2]
```diff
@@ -177,30 +177,36 @@ Proguardをお使いの場合、Proguardのルールファイルに下記のル
...
+   - Pass the `VirtusizeProduct` object to the `Virtusize.load` function
+   
    Kotlin
-
+   
    ```kotlin
-   (application as App)
+   val product = VirtusizeProduct(
+       externalId = "vs_dress",
+       imageUrl = "http://www.image.com/goods/12345.jpg"
```
Comment (gpark-vs) [inline README-JP.md:193]: Misalignment of three lines..? 🤔

File: README-JP.md [hunk 3]
```diff
@@ -177,30 +177,36 @@ Proguardをお使いの場合、Proguardのルールファイルに下記のル
...
+)
```
Comment (gpark-vs) [inline README-JP.md:194]: Misalignment..? 🤔

**Review by gpark-vs — COMMENTED** [review]
I have commented few misc things!
Please check them out!

**Comment by akueisara** [comment]
> I have commented few misc things!
> Please check them out!

@gpark-vs Thank you! I've made changes regarding your comments.

**Pull request merged**